### PR TITLE
now possible to use aggregators with localnames, in viewing mode (fn.…

### PR DESCRIPTION
**Pull request by aquadk**
now possible to use aggregators with localnames, in viewing mode (fn.pivot)

### CODE REVIEW

**Comment by aquadk** [comment]
Danish language added

File: dist/pivot.js [hunk 1]
```diff
@@ -907,7 +907,7 @@
     /*
     Pivot Table core: create PivotData object and call Renderer on it
      */
-    $.fn.pivot = function(input, opts) {
+    $.fn.pivot = function(input, opts, overwrite, locale) {
```
Comment (nicolaskruchten) [inline dist/pivot.js]: I'm not sure I understand what this modification aims to do...

File: dist/pivot.js [hunk 2]
```diff
@@ -925,6 +926,7 @@
         localeStrings: locales.en.localeStrings
       };
       opts = $.extend(defaults, opts);
+      opts.aggregator = opts.aggregators[opts.aggregatorName](opts.vals);
```
Comment (nicolaskruchten) [inline dist/pivot.js]: The API for `pivot()` assumes that the aggregator is given by the `aggregator` parameter, not looked up in an `aggregators` list, so this is a major API change, which I am not really in favour of.

File: dist/pivot.da.min.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+(function () {
```
Comment (nicolaskruchten) [inline dist/pivot.da.min.js]: Thanks for this. I will try to find time to port it into Coffeescript for inclusion.

**Comment by nicolaskruchten** [comment]
I've ported the DA locale file into coffeescript, thanks! :)

**Pull request closed without merging**